labs/sparknlp-ner/pbl_transform_script.scala

```diff
@@ -0,0 +1,90 @@
+import org.apache.spark.sql.{DataFrame, Dataset}
+import com.johnsnowlabs.nlp.annotator._
+import com.johnsnowlabs.nlp.annotators.ner.NerConverter
+import com.johnsnowlabs.nlp.base._
+import com.johnsnowlabs.util.Benchmark
+import com.sun.rowset.internal.Row
+import org.apache.spark.ml.Pipeline
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.functions.udf
+
+/*
+ * NOTE: This script is a standalone file to help you
+ * see the custom transformScala script written for
+ * this job. It is encoded as a .json string in
+ * process_ner_job.json file. Thus, making any edits
+ * on this file will not affect the job in anyways.
+ */
+def splitTokens(str: String): List[String] = {
+  val foo = str.split("@")
+  return foo.toList
+}
+
+def splitWords(str: String): (String, String) = {
+  // str is of format: word->London#result->LOC
+  val xs = str.split("#")
+  var first = ""
+  var secnd = ""
+  try {
+    first = xs(0).substring(6)
+    secnd = xs(1).substring(8)
+  } catch {
+    case e: Exception => {
+      first = "ERROR: " + str
+      secnd = "ERROR: " + str
+    }
+  }
+  (first, secnd)
+}
+
+def massageList(vals: List[String]): List[(String, String)] = {
+  for (e <- vals)
+    yield splitWords(e)
+}
```

Comment:
Instead of massageList and splitWords, do:

```
val regex = java.util.regex.Pattern.compile("word->(.*)#result->(.*)")
val tupes = words.map(w => {
  val m = regex.matcher(w)
  if (m.matches) {
    (m.group(1), m.group(2))
  } else {
    (w,"")
  }
}).filterNot(_._2.isEmpty)
```